packages/all/package.json

```diff
@@ -6,7 +6,7 @@
   "types": "esm/index.d.ts",
   "private": false,
   "scripts": {
-    "build": "rimraf ./component rimraf ./esm && tsc -p tsconfig.json && tsc -p tsconfig-cjs.json"
+    "build": "rimraf ./component && rimraf ./esm && tsc -p tsconfig.json && tsc -p tsconfig-cjs.json"
```

Comment:
```suggestion
    "build": "rimraf ./component & rimraf ./esm && tsc -p tsconfig.json & tsc -p tsconfig-cjs.json"
```

Reply:
separated the build into prebuild (for cleanup) and build